Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM ubuntu:20.04
+FROM pytorch/pytorch:latest
```

Comment:
We shouldn't use the pytorch docker images. And we also should never use `latest` as the docker file wont be reproducible.

Reply:
Funny, right now I'm chasing what looks like to be just that. I'll undo the docker changes and see how CI is. 

I ran that image and got things working locally but have been still having an issue reproducing things on one of our servers